assets/stylesheets/style.scss

```diff
@@ -1,6 +1,6 @@
 @import 'shared/utils';
 
-.wcc-root {
```

Comment:
Sorry I missed this. This solution would break horribly in Store-in-dotcom, because Calypso doesn't add the `wp-admin wp-core-ui` classes to the `</body>`, so no CSS would be applied.